cache/cache_test.go

```diff
@@ -3,6 +3,7 @@ package cache_test
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/bitrise-io/go-utils/env"
```

Comment:
NOTE: I suggest using `gci` linter to have a deterministic order for package imports.